vmdb/app/models/ems_refresh/save_inventory.rb

```diff
@@ -4,6 +4,7 @@ def save_ems_inventory(ems, hashes, target = nil)
     case ems
     when EmsCloud; save_ems_cloud_inventory(ems, hashes, target)
     when EmsInfra; save_ems_infra_inventory(ems, hashes, target)
+    when EmsContainer;   save_ems_container_inventory(ems, hashes, target)
```

Comment:
Align the method calls above to match `save_ems_container_inventory`.  And, probably just have one space for `save_ems_container_inventory`.

```ruby
  when EmsCloud;     save_ems_cloud_inventory(ems, hashes, target)
  when EmsInfra;     save_ems_cloud_inventory(ems, hashes, target)
  when EmsContainer; save_ems_cloud_inventory(ems, hashes, target)
```